test/base/automatic-speech-recognition.spec.ts

```diff
@@ -48,4 +42,16 @@ describe("AutomaticSpeechRecognition", () => {
       expect.any(Object),
     );
   });
+
+  it("should send a request with base64 audio", async () => {
```

Comment:
I guess this text is not accurate any more :) I think we always send FormData now (for certain types of models).